airbyte-integrations/connectors/destination-bigquery-denormalized/src/test/java/io/airbyte/integrations/destination/bigquery/formatter/GcsBigQueryDenormalizedRecordFormatterTest.java

```diff
@@ -248,8 +260,8 @@ public void testEmittedAtTimeConversion() {
     airbyteRecordMessage.setEmittedAt(1602637589000L);
     mockedFormatter.addAirbyteColumns(objectNode, airbyteRecordMessage);
 
-    assertEquals("2020-10-14 01:06:29.000000+00:00",
-        objectNode.get(JavaBaseConstants.COLUMN_NAME_EMITTED_AT).textValue());
+    assertEquals("1602637589000",
```

Comment:
why did this value change? Shouldn't we still be emitting ISO8601 timestamps?

Reply:
In the previous implementation this was testing `DefaultBigQueryDenormalizedRecordFormatter` but not `GcsBigQueryDenormalizedRecordFormatter` so if you would check `DefaultBigQueryDenormalizedRecordFormatterTest` there you could see this assert with expecting "2020-10-14 01:06:29.000000+00:00"

Reply:
wait, so the record formatters actually behave differently? that sounds like a bug, unless I'm misunderstanding?

(why do we need multiple record formatters anyway?)

Reply:
There is some difference between `DefaultBigQueryDenormalizedRecordFormatter` and `GcsBigQueryDenormalizedRecordFormatter`. For example in this case GCS s use MILLISECONDS but Default formatter MICROSECONDS. So the GCS just add small extra functionality on the top of default.

What I think is the tests for GCS formatter were designed wrong since it test the Default formatter functionality. What I propose is to leave tests for Default one as it is but adapt a bit GCS tests to cover it's own functionality only. Basically it's some replacement in json schema and this MILLISECONDS thing. 

What do you think?